api.py

```diff
@@ -44,83 +44,146 @@ async def status():
 
 @app.post("/validate")
 async def validate_schema_request_body(payload=Body(None)):
-    logger.info("Validating schema")
+    logger.info("Schema validation request received")
     return await validate_schema(payload)
 
 
 @app.get("/validate")
 async def validate_schema_from_url(url=None):
+    logger.debug("Attempting to validate schema from URL...", url=url)
     if url:
-        logger.info("Validating schema from URL", url=url)
         parsed_url = urlparse(url)
         domain = parsed_url.netloc
         if not is_domain_allowed(parsed_url, domain):
+            logger.warning(
+                "Schema validation request rejected - URL not allowed (this could be related to domain, base_url or repo_owner - see debug logs for values)",
+                url=url,
+            )
             return Response(
                 status_code=400,
                 content=f"URL domain [{parsed_url.hostname}] is not allowed",
             )
-
+        logger.info("Schema validation request accepted - URL allowed", url=url)
         try:
+            # Opens the URL and validates the schema
             with request.urlopen(parsed_url.geturl()) as opened_url:
                 return await validate_schema(data=opened_url.read().decode())
+            logger.info("Schema successfully validated from URL", url=url)
```

Comment:
I think this logger is unreachable as it is after the return

Reply:
This line was unreachable code. I have now removed this `info` log - if this log were to be output, it would be repeating information previously output in other logs, since the `validate_schema` function that would run before it includes a similar log.